cibuildwheel/macos.py

```diff
@@ -72,6 +72,9 @@ def install_cpython(version: str, url: str) -> Path:
 
     # if this version of python isn't installed, get it from python.org and install
     python_package_identifier = f'org.python.Python.PythonFramework-{version}'
+    python_executable = 'python3' if version[0] == '3' else 'python'
+    installation_bin_path = Path(f'/Library/Frameworks/Python.framework/Versions/{version}/bin')
```

Comment:
Minor nitpicking, but why are these moved? I think the reason they were only set later, is because they are only installed in that block. So logically, I think I'd prefer keeping it as it was, and setting them only after the package is installed (or checked that it is installed)

Reply:
because I use this values in line 89

Reply:
Oh, I didn't see the indentation in the diff, but it's still inside the if-statement. Pity, but OK.